app/src/main/res/values-sr/strings.xml

```diff
@@ -32,8 +31,11 @@
     <string name="contacts_desc">Тражи у мојим контактима</string>
     <string name="contacts_name">Контакти</string>
 
-    <string name="search_desc">Тражи на вебу помоћу Гугла</string>
-    <string name="search_name">Гугл претрага</string>
+    <string name="google_search_desc">FIXME Enable Google Web Search</string>
```

Comment:
Serbian translation was provided by @pejakm on #111, maybe he'll be able to help on this one?